testing/cli-e2etest/environment/manager.go

```diff
@@ -134,9 +134,15 @@ func (m *internalManager) Start(t *testing.T) {
 	require.NoError(t, err)
 	helpers.RequireExitCodeEqual(t, result, 0)
 
-	// TODO: think in a better way to assure readiness for Tracetest
-	// like https://golang.testcontainers.org/quickstart/ "Wait for Log" method
-	time.Sleep(readiness)
+	// wait until tracetest port is ready
+	waitForPort("11633")
+
+	if m.pokeshopEnabled {
+		// wait for pokeshop services
+		waitForPort("8081")
+		waitForPort("8082")
+		time.Sleep(10 * time.Second)
```

Comment:
Even with `waitForPort` for some reason Pokeshop drops the connection for a while, this is why I added `time.Sleep` again.